packages/react-hooks/src/inbox/middleware.ts

```diff
@@ -10,7 +10,7 @@ export default (api) => (store) => (next) => async (action) => {
   const state = store.getState();
 
   switch (action.type) {
-    case "root/PAGE_VISIBLE":
+    //case "root/PAGE_VISIBLE":
```

Comment:
i think the page visible, which was leaving a tab and coming back may have caused more issues that it solved